cdap-common/src/main/java/co/cask/cdap/common/logging/LoggingContextAccessor.java

```diff
@@ -51,6 +51,13 @@ public static void setLoggingContext(LoggingContext context) {
   }
 
   /**
+   * Removes the logging context
+   */
+  public static void resetLoggingContext() {
+    loggingContext.remove();
```

Comment:
@chtyim I don't see MDC being added to logs after resetting the logging context. Do we know why we need this code segment? Can I remove it?
    try {
      // Also copy context tags to MDC
      MDC.setContextMap(context.getSystemTagsAsString());
    } catch (IllegalStateException e) {
      // MDC will throw this if there is no valid binding. Normally this should happen, but in case it does,
      // we'll just ignore it as it doesn't affect platform logic at all as we always use loggingContext.
    }